### PR TITLE
remove kyrgyz translation for ckeditor

### PR DESCRIPTION
**Pull request by m4ra**
**Tasks**
- [ ] PR name contains story or task reference
- [ ] Documentation (docs and inline)
- [ ] Tests (including n+1 and django_assert_num_queries where applicable)
- [x] Changelog

### CODE REVIEW

File: apps/users/utils.py [hunk 1]
```diff
@@ -9,4 +9,4 @@ def set_session_language(user_email, language=None, request=None):
         language = User.objects.get(email=user_email).language
     activate(language)
     if hasattr(request, "session"):
-        request.session[LANGUAGE_SESSION_KEY] = language
+        request.session[settings.LANGUAGE_COOKIE_NAME] = language
```
Comment (goapunk) [inline apps/users/utils.py:12]: does this still do what we want it to do? django deprecated storing the language in the session and uses a cookie, so this code doesn't affect any django internals and I can't find any code from us reading the value.

Comment (m4ra) [reply apps/users/utils.py:12]: it is imported in apps/accounts/views.py
but you are right, cookie should be used on the browser. will remove the request part then

[the remark this answers is not in the document]
Comment (m4ra) [reply apps/users/utils.py:12]: The example in the link is to be used within the view method, when the HttpResponse is sent, but we don't have access to the response from outside the view.

[the remark this answers is not in the document]
Comment (goapunk) [reply apps/users/utils.py:12]: should we just remove the kyrgyz then and think about a solution for the deprecation as part of the django upgrade?

[the remark this answers is not in the document]
Comment (m4ra) [reply apps/users/utils.py]: I found a way to access the response, since this is a template based view. Have a look.

**Review by goapunk — APPROVED** [review]
LGTM, we can fix the language stuff in the django upgrade pr

**Pull request merged**